src/table/components/TableHeadWrapper.tsx

```diff
@@ -39,7 +39,11 @@ function TableHeadWrapper({
           // when nebula does not handle keyboard navigation
           const tabIndex = columnIndex === 0 && !keyboard.enabled ? 0 : -1;
           const isCurrentColumnActive = layout.qHyperCube.qEffectiveInterColumnSortOrder[0] === column.dataColIdx;
-          const handleKeyDown = (evt) => {
+          const ariaSort = isCurrentColumnActive
+            ? (`${column.sortDirection}ending` as 'ascending' | 'descending')
+            : undefined;
+
+          const handleKeyDown = (evt: React.KeyboardEvent) => {
```

Comment:
```suggestion
          const handleKeyDown = (evt: React.KeyboardEvent) => {
           const ariaSort = isCurrentColumnActive
            ? (`${column.sortDirection}ending` as 'ascending' | 'descending')
            : undefined;
```
ariaSort is only used inside handleKeyDown right?

Reply:
Nope, it is prop on TableCell